src/index.css

```diff
@@ -0,0 +1,307 @@
+/* vars and themes */
+:root {
+  --background: #ffffff;
+  --subForm: #f4f4f4;
+  --formInSubForm: #fffdfd;
+  --cds-text-primary: #161616;
+  --cds-text-secondary: #525252;
+  --cds-field: #f4f4f4;
+  --cds-border-strong: #8d8d8d;
+  --blue: #0f62fe;
+  --red: #da1e28;
+}
+
+.dark {
```

Comment:
description not readable in dark mode

<img width="932" alt="Screen Shot 2023-03-17 at 11 46 42 AM" src="https://user-images.githubusercontent.com/11401353/225992094-c7b17fb2-2e06-4f3d-ab9a-536528a8f0cf.png">
<img width="933" alt="Screen Shot 2023-03-17 at 11 47 21 AM" src="https://user-images.githubusercontent.com/11401353/225992247-498d5f59-a905-4b50-b8b7-f46c376fa82b.png">